src/home/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Navbar from '../navbar'
+
+function Home(){
+    return(
+     <Navbar/>
+    );
+}
```

Comment:
Intenta usar espaciados entre los delimitadores y símbolos de JavaScript para facilitar la legibilidad. [Acá](https://www.w3schools.com/js/js_conventions.asp) puedes encontrar una guía que te da más detalles
```suggestion
function Home() {
  return (
    <Navbar/>
  );
}
```

Reply:
Gracias Kike, ya corregí los espacios.